results-processor/main.py

```diff
@@ -12,13 +13,40 @@
 
 
 APPENGINE_INTERNAL_IP = '10.0.0.1'
+LOCK_FILE = '/tmp/results-processor.lock'
```

Comment:
Should the `Dockerfile` contain any explicit permissions setting to ensure that the user running `main.py` has permission to write to this path?

Reply:
Every user should be able to access `/tmp` on Linux. (I think it's required by [LSB](https://en.wikipedia.org/wiki/Linux_Standard_Base) but can't find the section right now.)